keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -65,6 +69,24 @@ class ArtifactController(
     @PathVariable type: ArtifactType
   ): List<String> =
     repository.artifactVersions(name, type)
+
+  @GetMapping(
```

Comment:
This was added for debugging purposes but it might be useful to leave it here -- let me know if it makes sense.

Reply:
I personally find these sorts of debugging REST API endpoints useful.